<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartNewEmployee</class>
 <widget class="QDialog" name="StartNewEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Employee</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtFName"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Other Names:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLineEdit" name="txtOtherNames"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ID No:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtIDNo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>PIN No:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtPINNo"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>NSSF No:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>NHIF No:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLineEdit" name="txtNHIFNo"/>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QLineEdit" name="txtNSSFNo"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdSaveEmployee">
       <property name="text">
        <string>Save Employee</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
